### PR TITLE
fix segmentation fork when Capella for epoch is MaxUint64

### PR DESCRIPTION
**Pull request by qu0b**
**What type of PR is this?**
Bug fix

**Which issues(s) does this PR fix?**
Fix segmentation fault when generating an interop genesis configuration for `CapellaForkEpoch != math.MaxUint64` and `cfg.DenebForkEpoch != math.MaxUint64`

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Hi @qu0b, thanks for the pull request. Could you elaborate on the error? How did this cause a segmentation fault?

**Comment by prestonvanloon** [comment]
Nevermind, I see that it's dereferencing a nil pointer!

File: cmd/prysmctl/testnet/generate_genesis.go [hunk 1]
```diff
@@ -274,10 +274,17 @@ func generateGenesis(ctx context.Context) (state.BeaconState, error) {
...
+		if gen.Config.ShanghaiTime != nil {
+			log.
+				WithField("shanghai", fmt.Sprintf("%d", *gen.Config.ShanghaiTime)).
+				Info("setting fork geth times")
+		}
+		if gen.Config.CancunTime != nil {
+			log.
+				WithField("cancun", fmt.Sprintf("%d", *gen.Config.CancunTime)).
+				Info("setting fork geth times")
+		}
```
Comment (prestonvanloon) [inline cmd/prysmctl/testnet/generate_genesis.go:287]:
```suggestion
		fields := logrus.Fields{}
		if gen.Config.ShanghaiTime != nil {
			fields["shanghai"] = fmt.Sprintf("%d", *gen.Config.ShanghaiTime)
		}
		if gen.Config.CancunTime != nil {
			fields["cancun"] = fmt.Sprintf("%d", *gen.Config.CancunTime)
		}
		log.WithFields(fields).Info("Setting fork geth times")
```

File: cmd/prysmctl/testnet/generate_genesis.go [hunk 2]
```diff
@@ -274,10 +274,17 @@ func generateGenesis(ctx context.Context) (state.BeaconState, error) {
 		gen.Config.ShanghaiTime = interop.GethShanghaiTime(f.GenesisTime, params.BeaconConfig())
 		//gen.Config.CancunTime = interop.GethCancunTime(f.GenesisTime, params.BeaconConfig())
```
Comment (prestonvanloon) [inline cmd/prysmctl/testnet/generate_genesis.go:275]:
```suggestion
```

We could delete this while you are here, please

**Comment by qu0b** [comment]
Thanks for the improvements, merged them

**Pull request merged**